ProjFS.Mac/PrjFSKextTests/KextMockUtilities.hpp

```diff
@@ -8,7 +8,11 @@ namespace KextMock
     struct PlaceholderValue
     {};
     
-    // Allow any value to match
+    // Matches with any parameter value
+    // ParameterPlaceholderValue can be used:
```

Comment:
Unless you're renaming the class to `ParameterPlaceholderValue` all of the `ParameterPlaceholderValue` in the comment should be replaced with `PlaceholderValue`.